cg/constants/delivery.py

```diff
@@ -78,9 +78,9 @@
 
 CLINICAL_DELIVERY_TAGS: list[set[str]] = [{HermesFileTag.CLINICAL_DELIVERY}]
 
-FASTQ_ANALYSIS_CASE_TAGS: list[set[str]] = []
+RAW_DATA_ANALYSIS_CASE_TAGS: list[set[str]] = []
 
-FASTQ_ANALYSIS_SAMPLE_TAGS: list[set[str]] = [
+RAW_DATA_ANALYSIS_SAMPLE_TAGS: list[set[str]] = [
```

Comment:
But these tags will only be used for cases with data_delivery=fastq, right? Maybe we can leave them as is?

Reply:
Right now this will be in line with the other workflows. i..e fastq will be raw data.